dnf.spec

```diff
@@ -38,18 +38,18 @@
 %endif
 
 # configurable name for the compat yum package
-%global yum_subpackage_name %{name}-yum
+%global yum_subpackage_name yum
+
+# provide %{name}-yum on fedora < 31 to avoid conflict with existing yum
+%if 0%{?fedora} && 0%{?fedora} < 31
+    %global yum_subpackage_name %{name}-yum
+%endif
```

Comment:
Drop this block, as this breaks further logic in the package.

Reply:
Can you elaborate? The logic should be equivalent to what you're proposing in the below hunk. I just wanted to make it explicit that `yum` is the default name for the compat subpackage and the other two are alternatives.

Reply:
Oh I see, you're forcing it to emit `yum` by default and overriding instead. This is fine then.

Reply:
Yeah, it really is just cosmetic, but I liked this better :)